app/src/main/java/fr/free/nrw/commons/CommonsApplication.java

```diff
@@ -43,6 +43,7 @@
 import fr.free.nrw.commons.logging.FileLoggingTree;
 import fr.free.nrw.commons.logging.LogUtils;
 import fr.free.nrw.commons.upload.FileUtils;
+import fr.free.nrw.commons.upload.structure.depicts.DepictDao;
```

Comment:
Please rename to `depictions.DepictionDao`, thanks!

Reply:
modified in the next commit